app/Core/Database.php

```diff
@@ -29,7 +29,9 @@ public static function getConnectionData () {
 	 * @return bool database status flag
 	 */
 	public static function initialize () : bool {
-		self::$database = new \mysqli ('localhost:3306', 'root', 'glitch123');
+		$config_data = parse_ini_file("../app/serverconfig.ini");
```

Comment:
Follow the code patterns displayed in this project; there should be a space between the function and the parenthesis.

Reply:
Also use single quotes; double quotes cause php to attempt to parse variables inside the string.

Reply:
done